lib/src/worker_protocol.pb.dart

```diff
@@ -13,12 +13,13 @@ class Input extends GeneratedMessage {
   static final BuilderInfo _i = new BuilderInfo('Input')
```

Comment:
I don't expect this to change often (if ever), and if its not formatted then it prevents me from just doing `dartfmt -w .` at the whole project level.